demo/make_local_dataset_app.py

```diff
@@ -0,0 +1,214 @@
+import shutil
+from pathlib import Path
+from typing import Tuple
+import gradio as gr
+import soundfile as sf
+import pandas as pd
+from huggingface_hub import hf_hub_download
+
+from speech_to_text_finetune.hf_utils import get_available_languages_in_cv
+
+parent_dir = "local_data"
+cv_dir = f"{parent_dir}/cv"
+custom_dir = f"{parent_dir}/custom"
+dataset_id_cv = "mozilla-foundation/common_voice_17_0"
+
+languages_dict = get_available_languages_in_cv(dataset_id_cv)
```

Comment:
This requires to log to the huggingface hub (was missing that in the instructions). 

Is it strictly required to finetune with a custom local dataset?

Reply:
>This requires to log to the huggingface hub (was missing that in the instructions).

You also have to manually request authorization to access the dataset through the HF web

Reply:
True, its not necessary to do the HF steps if you are doing custom dataset! I ll fix this. I am also hoping to use the Common Voice API to download CV data so I can completely get rid of the HF login + approval dependency